src/themes/masschallenge/modules/modal.overrides

```diff
@@ -19,3 +19,31 @@
 .ui.modal > .teal.segment {
   background-color:  #b8dde4 !important;
 }
+
+.ui.modal.centered {
+  margin-top: 4rem; 
+  max-width: 368px; 
+  min-width: 320px;
```

Comment:
I think this class pairing is over-specifying the width property. Adding `.centered` should, in my mind, center the modal (although it is unclear if that is vertically or horizontally by the class name alone). Since this is about addressing the width issues for a specific implementation of `.ui.mini.modal.centered`, it should include `.mini` so as to not adversely impact the widths of all other size modals. It may also be necessary to add a more specific class or id that makes it clear this change should only exist for modals that employ `.rbc-calendar`